defaults/clades.tsv

```diff
@@ -154,16 +153,12 @@ clade	gene	site	alt
 21L (Omicron)	nuc	28882	A
 21L (Omicron)	nuc	21618	T
 21L (Omicron)	nuc	22775	A
-21L (Omicron)	nuc	23525	T
 21L (Omicron)	nuc	23599	G
-21L (Omicron)	nuc	24424	T
-21L (Omicron)	nuc	27259	C
 
 21M (Omicron)	nuc	8782	C
+21M (Omicron)	nuc	10449	A
```

Comment:
To prevent inversion of clade labels we've started to include all ancestral mutations in the children - so this line `21M (Omicron) nuc 10449 A` should be added to 21L and 21K according to the rule.

Probably nothing will happen if we don't do this but it served us well with Delta.

Soon I'll also rewrite clades.tsv to take advantage of our new inheritance feature that was released in Augur a while ago. Now everyone should have had a chance to update to that version.

Reply:
Got it. I think this may have been the issue with https://nextstrain.org/staging/ncov/open/trial/clades-update/global with 21M appears as subclade of 21K. I'll revise as you suggest and confirm that this fixes the issue.